docs/azmcp-commands.md

```diff
@@ -48,6 +48,23 @@ azmcp cosmos database container item query --subscription <subscription> \
                        [--query "SELECT * FROM c"]
 ```
 
+### Data Explorer Operations
+```bash
+# List Data Explorer clusters in a subscription
+azmcp data-explorer cluster list --subscription <subscription>
+
+# Get details for a Data Explorer cluster
+azmcp data-explorer cluster get --subscription <subscription> --cluster-name <cluster-name>
+
+# List databases in a Data Explorer cluster
+azmcp data-explorer database list [--cluster-uri <cluster-uri> | --subscription <subscription> --cluster-name <cluster-name>]
+
+# Query a Data Explorer database
+azmcp data-explorer query [--cluster-uri <cluster-uri> | --subscription <subscription> --cluster-name <cluster-name>]
```

Comment:
It looks to me the implementation is azmcp data-explorer query --subscription <subscription> [--cluster-uri <cluster-uri> |  --cluster-name <cluster-name>] ?

Reply:
Yes, I see that, let's have explicit naming conversation and get aligned with product desires?  Ideally we match what is in the Azure CLI to not confuse the model and help them attach the two.

Reply:
The underlying sdk needs a cluster uri. It _does not_ need a subscription.
The question here is how to align with azure-mcp conventions.
If we want to align with existing implementation @xiangyan99 is correct, the "optional" part is the service identifier.

Reply:
In my view regarding //build, let's make it simple. Let's either eliminate <cluster uri> or require a subscription, regardless of whether <cluster uri> or <cluster name> is being used.

@jongio how do you think?

Reply:
I fixed my comment from above because it was misleading. the control plane sdk (the one used in az cli) does need a sub, but the data plane one doesn't, and both are used here.
AFAIC, for consistency, we can force it for initial version. @xiangyan99 / @jongio  your call here.